Storage/Storage/Model/Order+CoreDataProperties.swift

```diff
@@ -45,15 +45,14 @@ extension Order {
     @NSManaged public var shippingState: String?
     @NSManaged public var shippingTax: String?
     @NSManaged public var shippingTotal: String?
-    @NSManaged public var status: String
     @NSManaged public var total: String?
     @NSManaged public var totalTax: String?
     @NSManaged public var exclusiveForSearch: Bool
     @NSManaged public var coupons: Set<OrderCoupon>?
     @NSManaged public var items: Set<OrderItem>?
     @NSManaged public var notes: Set<OrderNote>?
     @NSManaged public var searchResults: Set<OrderSearchResults>?
-
+    @NSManaged public var status: String?
```

Comment:
Is this how you add a relationship to this? status is a one-to-one, not a one-to-many, so I'm not sure if it's considered a set.